src/App.js

```diff
@@ -1,23 +1,41 @@
-import logo from './logo.svg';
 import './App.css';
+import FoodBox from './components/FoodBox';
+import { Row } from 'antd';
+import AddFoodForm from './components/AddFoodForm';
+import { useState } from 'react';
+import foods from "./foods.json"
+import { v4 as uuidv4 } from 'uuid'
 
 function App() {
+  foods = foods.map((food) => {
+    const id = uuidv4();
+    food.id = id;
+    return food;
+  });
+
+  const [foodList, setFoodList] = useState(foods);
+  const [filteredFoodList, setFilteredFoodList] = useState(foodList);
+
+
+  function addNewFood(newFood) {
+    const newFoodList = [...foodList, newFood]
+    setFoodList(newFoodList)
+  }
+
+  function deleteHandler(id) {
+    const deleteResults = filteredFoodList.filter((food) => !(food.id === id));
+    setFilteredFoodList(deleteResults);
+  }
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <AddFoodForm addNewFood={addNewFood} />
+
+      <h1>Food List</h1>
+      <Row>
+        {filteredFoodList.map((food) => {
+          return <FoodBox food={food} deleteHandler={deleteHandler}></FoodBox>;
```

Comment:
Remember when we use map the highest component should have a key prop in order to react know which node its creating and have the correct reference
```suggestion
          return <FoodBox food={food} deleteHandler={deleteHandler} key={food.name}></FoodBox>;
```